sig/line/bot/v2/channel_access_token/api/channel_access_token_client.rbs

```diff
@@ -48,7 +54,11 @@ module Line
             grant_type: String, 
             client_id: String, 
             client_secret: String
-          ) -> [(IssueShortLivedChannelAccessTokenResponse|ErrorResponse), Integer, Hash[String, String]]
+          ) -> (
+            [IssueShortLivedChannelAccessTokenResponse, 200, Hash[untyped, untyped]] # when HTTP status code is 200
+            | [ErrorResponse, 400, Hash[untyped, untyped]] # when HTTP status code is 400
```

Comment:
This definition is also somewhat verbose, but it makes it clearer which type is returned for each HTTP status code.